notebooks/0-the-cloud/0-the-cloud.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p><a href="https://docs.astropy.org/en/stable/coordinates/" target="_blank">SkyCoord</a> will accept sexigesimal coordinates, so I wonder if this conversion is totally necessary. Maybe we can omit this for now</p><p> </p><p>I would also stress that unless the user has a specific reason for using query_region() [like a target name that doesn't resolve], they should probably use the other functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>okay, sounds good. Updated text to omit conversion and updated code to use query_criteria</p>